query/search.js

```diff
@@ -120,25 +120,14 @@ function generateQuery( clean ){
 }
 
 function getQuery(vs) {
-  if (isSingleFieldGeoambiguity(vs) && !hasQueryOrAddress(vs)) {
-    // return `undefined` for now until we exorcise the geodisambiguation demons
-    return;
-  } else {
```

Comment:
For readability, it might be nice to leave the else, with an explicit `return undefined;` and a reminder comment of why it's happening.